newton/_src/utils/schema_resolver.py

```
@@ -0,0 +1,642 @@
+        },
+    }
+
+
+def _solref_to_stiffness(solref):
+    """Convert MuJoCo solref (timeconst, dampratio) to internal stiffness.
+
+    k = 1 / (timeconst^2)
+    """
+    try:
+        timeconst = float(solref[0])
+    except Exception:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)